addons/medical_engine/script_macros_medical.hpp

```diff
@@ -179,6 +179,7 @@
 #define GET_OPEN_WOUNDS(unit)       (unit getVariable [VAR_OPEN_WOUNDS, []])
 #define GET_BANDAGED_WOUNDS(unit)   (unit getVariable [VAR_BANDAGED_WOUNDS, []])
 #define GET_STITCHED_WOUNDS(unit)   (unit getVariable [VAR_STITCHED_WOUNDS, []])
+#define GET_DAMAGE_THRESHOLD(unit)  (unit getVariable [QEGVAR(medical,unitDamageThreshold), [EGVAR(medical,AIDamageThreshold),EGVAR(medical,playerDamageThreshold)] select (isPlayer unit)])
```

Comment:
```suggestion
#define GET_DAMAGE_THRESHOLD(unit)  (unit getVariable [QEGVAR(medical,damageThreshold), [EGVAR(medical,AIDamageThreshold), EGVAR(medical,playerDamageThreshold)] select isPlayer unit])
```

I think if the expected value is different changing the name is fine, and the "unit" part can be omitted.

Reply:
Yeah i also was thinking to just set it to `ace_medical_damageThreshold`. It seems very close to the original one tho. So I am not sure.